### ISSUE TITLE
support-logging -- eliminate failure test for logs exceeding limit

### ISSUE THREAD

**Issue by tsconn23** [edgexfoundry/blackbox-testing#83]
This test is obsolete once the following PR is merged into edgex-go:

<img width="635" alt="screen shot 2018-10-02 at 2 24 53 pm" src="https://user-images.githubusercontent.com/1342980/46372312-680a7800-c650-11e8-95d5-2e56204044e9.png">

https://github.com/edgexfoundry/edgex-go/pull/617

This is because a change had been made whereby if the specified limit to the number of log messages (say, 1000) is greater than the configured ReadMaxLimit (say, 100) then the parameter supplied by the user will be changed to agree with ReadMaxLimit. Therefore an error to say there are more logs available than the user specified will never be thrown.

**Comment by weichou1229** [comment]
Rewrite test to check log messages size, which not exceed the configured ReadMaxLimit.
